### PR TITLE
[DOCS-6221] Add Send and Archive info in Outlook

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM

**Review by gravitonian — COMMENTED** [review]
Built and reviewed the different pages with additions related to the new Send & Archive action. Looks ok

File: microsoft-outlook/latest/config/index.md [hunk 1]
```diff
@@ -985,6 +1009,7 @@ Use this file to set up attributes and metadata settings.
     |`storeLink`|Controls the Archive as link email option|`true`: email is replaced with a link to email stored in Alfresco<br><br>`false`: email is not replaced with a link to the email stored in Alfresco. This is the default setting.|
     |`storeMsg`|Controls the Store original Outlook .MSG file archive option|`true`: original Outlook . MSG file is stored on upload to Alfresco<br><br>`false`: original Outlook . MSG file is not stored on upload to Alfresco. This is the default setting.|
     |`compress`|Controls the Compress message while uploading setting|`true`: message is compressed while uploading to Alfresco. This is the default setting.<br><br>`false`: message is not compressed while uploading to Alfresco|
+    |`alwaysUseDefaultSendAndArchiveFolder`|Controls the Always use default folder archive option|`true`: uses the specified default folder. This is the default setting.<br><br>`false`: select folder with every upload.|
```
Comment (mhofmann-itnovum) [inline microsoft-outlook/latest/config/index.md:1012]: false is the default here.

Comment (anxumalo) [reply microsoft-outlook/latest/config/index.md:1012]: Thanks for the confirmation - the fix is in PR #219

**Pull request merged**